docs/rules/array-callback-return.md

```diff
@@ -72,6 +72,21 @@ var foo = Array.from(nodes, function(node) {
 var bar = foo.map(node => node.getAttribute("id"));
 ```
 
+## Options
+
+This rule has an object option:
+
+* `"allowImplicit": false` (default) disallows implicitly returning `undefined` with a `return` statement.
```

Comment:
I understand that the description is speaking about when this option is set to false, but I can imagine how someone else reading this for the first time might not read it that way. For clarity, can we update it so something like the following?

* `"allowImplicit": false` (default) When set to `true`, allows implicitly returning `undefined` with a `return` statement.